atc/worker/worker.go

```diff
@@ -549,11 +549,11 @@ func (worker *gardenWorker) cloneLocalVolumes(
 		inputVolume, err := worker.volumeClient.FindOrCreateCOWVolumeForContainer(
```

Comment:
Note: We do a COW specific find or create here. This ensures that a parent handle gets stored in the child volumes db row. I don't think there's any side effects if we use an import strategy to copy over but still consider returned `inputVolume` a child.